src/gui/folderman.cpp

```diff
@@ -945,11 +945,15 @@ void FolderMan::runEtagJobIfPossible(Folder *folder)
 
 void FolderMan::slotAccountRemoved(AccountState *accountState)
 {
+    QVector<Folder *> foldersToRemove;
     for (const auto &folder : qAsConst(_folderMap)) {
         if (folder->accountState() == accountState) {
-            folder->onAssociatedAccountRemoved();
```

Comment:
why do you remove this call ?
sounds like we were doing some stuff with the VFS ?

Reply:
@mgallien Yes, we're doing it here. But, we're also doing it in `void Folder::wipeForRemoval()`, so, this is a duplicate, and, the worst thing, it leads to a crash in CfAPI unregister code because of calling `_vfs->stop();` and `_vfs->unregisterFolder();` 2 times, while we only need to call it once. And now, with my other change, it is all being called within the  `void Folder::wipeForRemoval()`, which is just enough.

Reply:
@mgallien And one more detail, `void FolderMan::removeFolder(Folder *f)` will call `void Folder::wipeForRemoval()` always